### DISCUSSION TITLE
为什么音乐正常播放进度条和歌词不进行滚动

### DISCUSSION THREAD

**Discussion by W-ang-nan** [SevenOutman/aplayer-react#27] [Q&A]
<APlayer
      audio={{
        name: title,
        artist: { name: title },
        url: url,
        cover: img,
        lrc: lrc,
      }}
      // appearance="fixed"
      initialLoop="one"
      initialOrder="random"
      autoPlay
    />

**Comment by SevenOutman** [comment]
你的歌词是具有时间标记的 lrc 格式吗？例如：![image](https://github.com/SevenOutman/aplayer-react/assets/8225666/cf5d1f8e-586d-420e-8e14-db431bf98f76)
